lib/transforms/addPushManifest.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Adds a google app engine compatible push manifest file to the graph.
```

Comment:
FWIW, the push manifest is a convention I made up. It's not an app engine thing or support by app engine team in any way.  It's meant to be a generic format that you can consume on any h2 provider. We just happen to show   app engine examples because we know it well :)

Maybe just "Adds a google app engine" -> "Adds a http2 push manifest..."